### PR TITLE
Support to get Pipeline and PipelineRun List

### PR DESCRIPTION
**Pull request by LinuxSuRen**

### CODE REVIEW

File: pkg/job/blueocean.go [hunk 1]
```diff
@@ -89,6 +97,39 @@ func (c *BlueOceanClient) GetBuild(option GetBuildOption) (*PipelineBuild, error
 	return &pb, nil
 }
 
+// PipelineRun represents a PipelineRun of Jenkins BlueOcean
+type PipelineRun struct {
```
Comment (JohnNiang) [inline pkg/job/blueocean.go:101]: Please take a look at PipelineBuild:

https://github.com/jenkins-zh/jenkins-client/blob/8d848a9886c2aeceec09de5c147d4949c1f0e9c5/pkg/job/blueocean.go#L140-L167

I prefer to use it or let `PipelineRun` embed in `PipelineBuild`.

Comment (LinuxSuRen) [reply pkg/job/blueocean.go:101]: My bad. I think they are the same thing. We can use one of them. But I cannot see `PipelineBuild` was used in any BlueOcean documents.

Comment (JohnNiang) [reply pkg/job/blueocean.go:101]: You are right. In blueocean source code, it is called [PipelineRun](https://github.com/jenkinsci/blueocean-plugin/blob/master/blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineRunImpl.java).

Comment (LinuxSuRen) [reply pkg/job/blueocean.go:101]: So, I'm going to rename `PipelineBuild` to `PipelineRun` if you think it's ok.

Comment (JohnNiang) [reply pkg/job/blueocean.go:101]: Make sense to me

Comment (LinuxSuRen) [reply pkg/job/blueocean.go:101]: By the way, I'm wondering if it's possible that we can put this projects' link into [BlueOcean restful document](https://github.com/jenkinsci/blueocean-plugin/tree/master/blueocean-rest) once we covered enough APIs.

Comment (JohnNiang) [reply pkg/job/blueocean.go]: That would be nice. And I will follow this approach in further PR.

File: pkg/job/blueocean.go [hunk 2]
```diff
@@ -53,8 +53,8 @@ type BuildOption struct {
 }
 
 // Build builds a pipeline for specific organization and pipelines.
-func (c *BlueOceanClient) Build(option BuildOption) (*PipelineBuild, error) {
-	var pb PipelineBuild
+func (c *BlueOceanClient) Build(option BuildOption) (*PipelineRun, error) {
+	var pb PipelineRun
```
Comment (JohnNiang) [inline pkg/job/blueocean.go:57]: Please rename the variable name.

File: pkg/job/blueocean.go [hunk 3]
```diff
@@ -88,31 +88,15 @@ type GetBuildOption struct {
 }
 
 // GetBuild gets build result for specific organization, run ID and pipelines.
-func (c *BlueOceanClient) GetBuild(option GetBuildOption) (*PipelineBuild, error) {
-	var pb PipelineBuild
+func (c *BlueOceanClient) GetBuild(option GetBuildOption) (*PipelineRun, error) {
+	var pb PipelineRun
```
Comment (JohnNiang) [inline pkg/job/blueocean.go:92]: Same as above

**Review by JohnNiang — CHANGES_REQUESTED** [review]

**Comment by JohnNiang** [comment]
LGTM

**Pull request merged**